imagestreams/nodejs-rhel7.json

```diff
@@ -73,7 +73,7 @@
           "openshift.io/provider-display-name": "Red Hat, Inc.",
           "description": "Build and run Node.js 10 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/nodeshift/centos7-s2i-nodejs.",
           "iconClass": "icon-nodejs",
-          "tags": "builder,nodejs",
+          "tags": "builder,nodejs,hidden",
```

Comment:
@siamaksade looks like the `latest` tag points to the "hidden" RHOAR imagestream. Please update the `latest` tag to point to the `10-SCL` tag.

I'm not worried about breaking customers in this case - it's widely accepted that if you use a `latest` tag, you are accepting the risk that the image can change underneath you.

Reply:
Fixed